strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/users/UserController.java

```diff
@@ -106,9 +106,10 @@
     @PreAuthorize("hasAuthority('VIEW_USER')")
     @GetMapping(produces = { MediaType.APPLICATION_JSON_VALUE })
     @ResponseBody
-    public ResponseEntity getUsers()
+    public ResponseEntity getUsers(@RequestParam(value = "skip") Long skip,
+                                   @RequestParam(value = "limit") Integer limit)
```

Comment:
Thanks for mentioning me @sbespalov! 

This is actually a breaking change, because it requires the request parameters to be present (which they are not in the UI). Lets make them optional for the `API` and leave some reasonable defaults. This will make it more accessible for people who are playing around with the API and will preserve backwards compatibility.

```suggestion
    public ResponseEntity getUsers(@RequestParam(value = "skip", required = false, defaultValue = 0) Long skip,
                                   @RequestParam(value = "limit", required = false, defaultValue = 25) Integer limit)
```

A test for the `optional` values should be added as well (i.e. `/api/users`).

@anki2189 would you like to have a go with the UI which should have a? If not - we should add a follow-up task linked to this one to since I'm currently busy with another task and it will be a while until I can get to this.